windows_integration/README.md

```diff
@@ -0,0 +1,27 @@
+# Kanidm Windows Client
+Currently the client consists of one part, the Security Support Provider & Authentication Package (SSP/AP).
+
+## Building
+The only supported toolchain to build Kanidm's Windows Client is `x86_64-pc-windows-msvc` which requires the MSVC toolchain to be installed. You can get the toolchain with the Visual Studio Workload `Desktop Development with C++`. The toolchain is also available with the same workload in the [Visual Studio Build Tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022).
+
+## Installation
+In order for the client to function, all parts must be installed in the correct place and registry values set. In the future there will be a powershell script and maybe MSI package to automate this.
+
+## SSP/AP
+To install the SSP/AP follow the below steps:
+* Create the directory `C:\Program Files\kanidm`
+* Copy the file `kanidm_win_authlib.dll` to the newly created directory
+* Open the Registry Editor and navigate to `HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Lsa`
+* Add the path `C:\Program Files\kandim\kandim_win_authlib\0\0` (The \0 is a null character)
```

Comment:
"Security Packages"

Reply:
Kan-d-im is incorrect spelling, needs to be kan-i-dm.

Reply:
Get-EventLog -LogName Security -EntryType Error
Worth putting in the powershell version of the how to get event log to help debug the integration.

Reply:
Set-ItemProperty -Path "HKLM:\System\CurrentControlSet\Control\Lsa" -Name "Security Packages" -Value "C:\Program Files\kanidm\kanidm_win_authlib\0"

Reply:
Actually, I think you only need the \0 to seperate multiple dlls - because there is only one here you don't need it and I think that's what's throwing off the reg.